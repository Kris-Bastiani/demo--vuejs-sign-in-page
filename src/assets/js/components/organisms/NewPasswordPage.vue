<template lang="pug">
	app-page
		.container.flx_aln_y__center.pad__sml.w_max__reg
			.w__full.w__half__at_above_680
				h1.mrg_b__lrg New password

				new-password-form

			img.dsp__none__below_680.w__half(src='/assets/images/reset_pwd.png', alt='', role='presentation')
</template>

<script>
import AppPage from '../molecules/AppPage.vue';
import NewPasswordForm from '../molecules/NewPasswordForm.vue';

export default {
	components: {
		AppPage,
		NewPasswordForm,
	},
};
</script>
