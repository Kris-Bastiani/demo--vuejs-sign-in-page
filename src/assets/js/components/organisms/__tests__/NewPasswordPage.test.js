import 'regenerator-runtime/runtime';
import { shallowMount } from '@vue/test-utils';
import NewPasswordPage from '../NewPasswordPage.vue';

describe('NewPasswordPage', () => {
	it('Matches snapshot', () => {
		const wrapper = shallowMount(NewPasswordPage);
		expect(wrapper.element).toMatchSnapshot();
	});
});
