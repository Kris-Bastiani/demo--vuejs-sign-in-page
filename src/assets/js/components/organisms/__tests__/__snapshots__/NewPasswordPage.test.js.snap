// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewPasswordPage Matches snapshot 1`] = `
<app-page-stub>
  <div
    class="container flx_aln_y__center pad__sml w_max__reg"
  >
    <div
      class="w__full w__half__at_above_680"
    >
      <h1
        class="mrg_b__lrg"
      >
        New password
      </h1>
      <new-password-form-stub />
    </div>
    <img
      alt=""
      class="dsp__none__below_680 w__half"
      role="presentation"
      src="/assets/images/reset_pwd.png"
    />
  </div>
</app-page-stub>
`;
